docs/developers/contributing/documentation/index.md

```diff
@@ -28,6 +28,7 @@ This is documentation is written with [Jinja](https://jinja.palletsprojects.com/
 5. The [API reference documentation](api) is autogenerated from the napari source code docstrings.
 Docstrings are written in the [reStructuredText format](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html)
 and any modifications to them should be submitted to the [napari/napari](https://github.com/napari/napari) repository.
+6. Importantly, all of the dependencies for building the documentation are in the [the `pyproject.toml` of napari/napari](https://github.com/napari/napari/blob/main/pyproject.toml) under the optional dependencies `docs` and `gallery`.
```

Comment:
When should we use `gallery`?

Reply:
I think that's just got napari/napari examples. 
`gallery` is part of `docs` for building the gallery in napari/docs and part of `testing` for testing the examples in napari/napari
Do you think we should explain that here?

Reply:
the napari/napari pyproject toml dependency groups are super complex now with a lot of nesting. they probably need docs of their own at this point 😬

Reply:
Yeah, I'm not sure I understand. The napari/napari examples ARE the gallery so it sounds redundant? Unless it means

"If you only have napari/napari and want to run all the gallery examples, you need napari[gallery]"

Reply:
You should like almost never need to *install* with `[gallery]`, because you'd have either `testing`, `dev`, or `docs`. You don't even get the examples if you don't clone the repo... 
but `gallery` does contain dependencies of docs, which is what I was trying to explain here.

Reply:
OK what we need to update is:
https://napari.org/stable/developers/contributing/documentation/index.html#adding-examples-to-the-gallery
And here explain that any new dependencies to the examples go in `gallery`

Reply:
Can definitely be a follow-up!

Reply:
Already took care of it 😄